bin/compile

```diff
@@ -124,6 +124,6 @@ echo "$(exports "$TARGET_DIR")" > $BUILDPACK_DIR/export
 topic "Configure application environment"
 mkdir -p $BUILD_DIR/.profile.d
 exports "\$HOME/vendor/poppler" > $BUILD_DIR/.profile.d/000_poppler.sh
+echo "export PATH=/app/vendor/poppler/bin:$PATH" >> $BUILD_DIR/.profile.d/000_poppler.sh
```

Comment:
I believe you should be able to add this to the `exports` function, near the top of the file.

Reply:
I see, sorry :-(

Reply:
solved by my last commit. Thanks!